### ISSUE TITLE
Training data & I3D model training issue. 

### ISSUE THREAD

**Issue by 2000222** [trevphil/TechniqueAnalysis#16]
Hello, thanks for your sharing and I’m so interested in the video classifier subsection. 
1. If you are comfortable, could u please provide us the training dataset for the whole training pipeline? 
2. I’ve run the train_model.py on custom dataset, but I got some problems about the dimensions of logits. So I think it is good for me the explore the inner logits with your original training data. Thank u very much.

**Comment by 2000222** [comment]
I've solved the problem well~
